mockserver/mockserver_test.go

```diff
@@ -4,17 +4,20 @@ import (
 	"context"
 	"testing"
 
+	"github.com/btcsuite/btcd/btcjson"
 	"github.com/btcsuite/btcd/chaincfg/chainhash"
 	"github.com/filecoin-project/go-jsonrpc"
 	"github.com/stretchr/testify/assert"
 )
 
 type Client struct {
-	Ping             func(int) int
-	GetBestBlockHash func() (*chainhash.Hash, error)
-	GetBlockCount    func() (int64, error)
-	GetBlockHash     func(blockHeight int64) (*chainhash.Hash, error)
-	GetBlockHeader   func(blockHash *chainhash.Hash) (*BlockHeader, error)
+	Ping              func(int) int
+	GetBestBlockHash  func() (*chainhash.Hash, error)
+	GetBlockCount     func() (int64, error)
+	GetBlockHash      func(blockHeight int64) (*chainhash.Hash, error)
+	GetBlockHeader    func(blockHash *chainhash.Hash) (*btcjson.GetBlockHeaderVerboseResult, error)
```

Comment:
I not sure but https://developer.bitcoin.org/reference/rpc/getblockheader.html have one more field call verbose. Can you check this?

Reply:
Added `verbose bool` to function signature.